python/pyspark/sql/utils.py

```diff
@@ -121,7 +121,10 @@ def require_minimum_pandas_version():
     from distutils.version import LooseVersion
     try:
         import pandas
+        have_pandas = True
     except ImportError:
+        have_pandas = False
+    if not have_pandas:
```

Comment:
So, this is to make the exception change look clean? I think it's fine to leave as is for now. I guess here we'll lose the information about exactly where the exception happened in Python 3 if I haven't missed something.